src/components/vm/consoles/serial.tsx

```diff
@@ -109,12 +111,24 @@ export const SerialActive = ({
     );
 };
 
-export const SerialInactive = () => {
+export const SerialInactive = ({
+    vm,
+} : {
+    vm: VM,
+}) => {
     return (
         <EmptyState>
             <EmptyStateBody>
                 {_("Start the virtual machine to access the console")}
             </EmptyStateBody>
+            <EmptyStateFooter>
+                <Button
+                    variant="secondary"
+                    onClick={() => vmStart(vm)}
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2376-3aa4fc32-20251103-073120-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)